### ISSUE TITLE
block/010 fail

### ISSUE THREAD

**Issue by ChanChunSong** [osandov/blktests#71]
I got this error log when running run_fio_job.
fio: ENOSPC on laying out file, stopping 
And the null block doesn't support direct in my environment.
Is it necessary?

**Comment by osandov** [comment]
Yes, for this test case, direct is necessary. Without direct, we're just testing the page cache, not the block layer. If you're still seeing this failure, can you paste the full test output?
